op-chain-ops/cmd/celo-migrate/main.go

```diff
@@ -166,37 +177,26 @@ func main() {
 		Usage: "Migrate Celo block and state data to a CeL2 DB",
 		Commands: []*cli.Command{
 			{
-				Name:    "blocks",
-				Aliases: []string{"b"},
-				Usage:   "Migrate Celo block data to a CeL2 DB",
-				Flags:   blockMigrationFlags,
+				Name:    "pre",
+				Aliases: []string{"pre-migration", "p"},
```

Comment:
I'd advise against using aliases for commands, since it can just lead to confusion later on when there are multiple different ways of executing a command that does the same thing.

I would just stick to "pre" and "full" since the binary is called celo migrate. Then you can explain in the help for "full" that it either does an entire migration or that it can complete migration on a pre migrated datadir.